File: exercicios/Python/1113/1113.py
a = 100
b = 1000

while a != b:
    a,b = map(int,input().split())

    asc = 'Crescente'
    desc = 'Decrescente'

    if a < b:
        print(asc)
    if a > b:
        print(desc)
    if a == b:
        print('')

File: exercicios/JavaScript/1113/1113.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

while(true){
    var line1 = lines.shift().split(' ');

    var x = parseInt(line1[0]);
    var y = parseInt(line1[1]);

    if(x == y){
        break;
    }
    if(x > y){
        console.log("Decrescente");
    }else{
        console.log("Crescente");
    }
}